components/bl-modal-component/src/index.js

```diff
@@ -31,17 +31,19 @@ export default function ModalComponent({ component, eventHandlers, pods, elRef }
     }
   };
 
-  if (!visibility) {
+  if (!display) {
```

Comment:
```suggestion
  if (!display || !visibility) {
```

If I understand correct, in that case we could avoid using fragment